typedoc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I am not sure why the typedoc for a precise project should live outside it, are we planning to use typedoc for another project? in this case I imagine have a global config and a specific one on every project.